### ISSUE TITLE
Add Chronometer control

### ISSUE THREAD

**Issue by insinfo** [Microsoft/xaml-standard#173]
![chronometer 1](https://cloud.githubusercontent.com/assets/12227024/26517364/82704d7c-426d-11e7-86ed-96253d1e3958.jpg)
![chronometer 4](https://cloud.githubusercontent.com/assets/12227024/26517366/8271e092-426d-11e7-95ca-b92778ef59af.png)
![chronometer 3](https://cloud.githubusercontent.com/assets/12227024/26517365/827096a6-426d-11e7-90b0-626fe1485ed5.png)
![chronometer 2](https://cloud.githubusercontent.com/assets/12227024/26517367/8273fb7a-426d-11e7-8fd1-b5c061349c56.jpg)

**Comment by insinfo** [comment]
Yes it is true that it is something simple to implement, but we can make life easier for the developer because it is not ready to be used, that is what Google is doing.

**Comment by crutkas** [comment]
@insinfo what I saw was a variation of a text block for their implementation. What your showing above is a full blown UX. Is the ask here for parity with android?

https://developer.android.com/reference/android/widget/Chronometer.html

**Comment by birbilis** [comment]
In my view commonly needed controls should be modelled (PME - Property/Method/Event) in such a standard. MS seems to focus on just P and E here btw

**Comment by crutkas** [comment]
@birbilis. Start proposing methods on stuff you feel is needed. I am trying to help guide the conversation on important stuff versus variants of the same control that could be included in the base.

**Comment by birbilis** [comment]
What I mean is that common components' PME models should be defined using say C# interfaces (and interface-only [even multiple] inheritance) probably in a separate repo if that's out of scope

**Comment by mossyblog** [comment]
Why is this a still open issue?
